iati/core/resources/test_data/202/valid_sum.xml

```diff
@@ -1,6 +1,54 @@
 <?xml version="1.0"?>
-
+<!--given values sum to specified amount-->
 <root_element>
-  <recipient-country code="AF" percentage="50" /><!-- percentages sum to 100 correctly according to test ruleset -->
-  <recipient-region code="489" vocabulary="1" percentage="50" />
+  <condition></condition><!--condition case-->
+  <element1>3</element1><!--single path-->
+  <element2>1</element2><!--multiple paths-->
+  <element3>2</element3>
+  <element4>3</element4><!--duplicate paths-->
+  <element5>-500</element5><!--negative value-->
+  <element6>-500</element6>
+  <element7>50</element7><!--sum greater than standard percentage value-->
+  <element8>51</element8>
+  <element9>10</element9><!--sum to decimal value-->
+  <element10>5.5</element10>
+  <element11>0</element11><!--sum zero value-->
+  <element12>0</element12>
+  <element13>5e+99</element13><!--big value-->
+  <element14>5e+99</element14>
+  <element15>-5e+99</element15><!--tiny value-->
+  <element16>-5e+99</element16>
+  <element17>1498962.29</element17><!--exponential value-->
```

Comment:
This is a decimal value rather than an exponential one

Reply:
done